guides/integrate-nextauth.md

```diff
@@ -0,0 +1,427 @@
+
+```javascript
+import { Auth } from "@auth/core"
+import BeyondIdentity from "@auth/core/providers/beyondidentity"
+
+const request = new Request("https://example.com")
+const response = await Auth(request, {
+  providers: [BeyondIdentity({ clientId: "", clientSecret: "", issuer: "" })],
+})
+```
+
+</TabItem>
+</Tabs>
+
+We will overwrite the `Request` _url_ and `BeyondIdentity` _clientId_, _clientSecret_ and _issuer_ values later in this process, so you can use the dummy value for now.
+
+## Install the JavaScript SDK
+
+In order to use Beyond Identity functionality in your application, you will need to install the SDK. The [JavaScript SDK](https://github.com/gobeyondidentity/bi-sdk-js) provides functionality from passkey creation to passwordless authentication. A set of functions are provided to you through an `Embedded` namespace.
+
+## Initialize the JavaScript SDK
+
+Once you've installed the SDK, initialize it so that you can call the Embedded functions.
+
+<SetupJavaScript />
+
+## Set up Beyond Identity as an Identity Provider
+
+To set up Beyond Identity as an Identity Provider, you need to create a [Realm](/docs/v1/workflows/realms) to hold identities and configuration. Inside that realm, you'll need to create an [Application](/docs/v1/workflows/applications) that contains the authentication flow configuration. These can be configured in you admin console that was created for you when you signed up for a developer account.
+
+### Create a Realm
+
+From the admin console, click the realm drop-down, and click **Create new realm**.
+
+<Arcade clip={Clip.CreateRealm} />
+
+### Create an Application
+
+From the admin console, click **Apps**, then click **Add app**.
+
+There is a lot to configure when creating an application. When creating your application make sure:
+
+- Redirect URIs contains your application's App Scheme or Universal URL
+- Configuration Type (in the Authenticator Config tab) is set to `Embedded SDK`
+- Invoke URL (in the Authenticator Config tab) contains your application's App Scheme or Universal URL
+- Invocation Type (in the Authenticator Config tab) is set to `Automatic`.
+
+For help choosing options, visit the following guides:
+
+- [Workflows: Applications](/docs/v1/workflows/applications)
+- [Authenticator Config](/docs/v1/platform-overview/authenticator-config)
+
+<Arcade clip={Clip.CreateApplication} />
+
+Now that we have created our application, we are ready to update our values. Store these values in your Next application's environment variables to use with the Beyond Identity provider. 
+
+1. For `BeyondIdentity` _clientId_, copy and paste the value from Applications -> Your New Application -> External Protocol -> Client ID
+1. For `BeyondIdentity` _clientSecret_, copy and paste the value from Applications -> Your New Application -> External Protocol -> Client Secret
+1. For `BeyondIdentity` _issuer_, copy and paste the value from Applications -> Your New Application -> External Protocol -> Issuer
```

Comment:
I think the issuer part only makes sense for Auth.js. Is that the same as the well-known in NextAuth?

Reply:
Almost. Issuer is the base part of the URL,
`https://auth-REGION.beyondidentity.run/v1/tenants/TENANT/realms/REALM/applications/APPLICATION`, and Well-Known is the Issuer + `.well-known/openid-configuration` (also known as the Discovery Endpoint). These are still needed in a custom NextAuth provider, but I can separate these into their respective tabs

Reply:
Ok cool, yea I think different tabs would be helpful